### PR TITLE
11.0 mig website_sale_default_country

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa

### CODE REVIEW

File: website_sale_default_country/__manifest__.py
Comment (chienandalu) [inline website_sale_default_country/__manifest__.py]: https://github.com/OCA/e-commerce

**Review by chienandalu — COMMENTED** [review]
Functional test ok. Just some little comments.

- Try to use the new OCA readme system: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme
- Put yourself in contributors credits :wink:

**Review by rafaelbn — COMMENTED** [review]
Tested functionally :+1: 

Please review  codecov/patch — 57.14% of diff hit (target 100%)

**Comment by rafaelbn** [comment]
cc @Tecnativa

**Review by yajo — APPROVED** [review]
Approved when you attend https://github.com/OCA/e-commerce/pull/239#pullrequestreview-125491595, please.

Thanks!

**Comment by pedrobaeza** [comment]
Testing this is a total nightmare with current PhantomJS tour system, so leave it as is. Please attend chienandalu's comment about README + contribution (check other one I have just submitted in your PRs as example).

**Comment by yajo** [comment]
> Testing this is a total nightmare with current PhantomJS tour system

Actually it was a nightmare until odoo 9. From v10, the tour system works fine, and from v11 we even have a new testing system (I haven't used it yet, though, and it's mostly for widgets, but it's awesome).

The pain now comes from tests is not much different from the pain of writing tests we always have had.

Clarifying that, it's not like I'm asking for tests here; although it would be great indeed :blush:

**Review by chienandalu — APPROVED** [review]

**Comment by yajo** [comment]
@ernestotejeda this is ready to merge; please squash migration commits in 1 to do it. Thanks!

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**
